src/components/books/list.css

```diff
@@ -54,32 +109,126 @@
 }
 
 .form_title {
-  width: 10.875rem;
+  width: 15.875rem;
   height: 1.5rem;
   margin: 1.813rem 32.75rem 1.188rem 0;
   font-size: 1.25rem;
+  color: #888;
+  letter-spacing: -0.18px;
+  font-weight: bold;
+  font-family: 'Montserrat', sans-serif;
 }
 
 form {
-  width: 75vw;
+  width: 85rem;
   margin: auto;
 }
 
 .input_ctn {
   display: flex;
   flex-direction: row;
-  width: 75rem;
+  width: 85rem;
   align-items: center;
 }
 
```

Comment:
- [ ] Please try to make your content fit on the page when you open the app you can reduce some `padding` or `margin` or `width` check what is causing this
![re](https://user-images.githubusercontent.com/44745855/131920424-d70c4d5b-5ad4-4142-979f-189cf89a205d.png)